code/modules/food_and_drink/snacks.dm

```diff
@@ -2235,22 +2234,15 @@ var/list/valid_jellybean_reagents = childrentypesof(/datum/reagent)
 	return
 
 /obj/item/kitchen/everyflavor_box/examine()
```

Comment:
The original code is fairly scuffed, I removed the part that modifies the amount of beans left in the bag when someone examines the bag, and the part where donuts are counted as beans.